### PR TITLE
[ISSUE #3260] Added Null ptr excep to method

### LINKED ISSUE #3260: [Enhancement] Method invocation 'getBytes' may produce 'NullPointerException'[EventMeshTestUtils]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java line 180,195

### Describe the solution you'd like

image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by nikam14**
Made changes in eventmesh-examples.
Added null pointer exception to method invocation 'getbytes'. Open for help and feel free to add any thing.
Fixes #3260

### CODE REVIEW

File: eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java [hunk 1]
```diff
@@ -192,7 +193,7 @@ public static CloudEvent generateCloudEventV1SyncRR() {
             .withSource(URI.create("/"))
             .withDataContentType(ExampleConstants.CLOUDEVENT_CONTENT_TYPE)
             .withType(EventMeshCommon.CLOUD_EVENTS_PROTOCOL_NAME)
-            .withData(JsonUtils.toJSONString(content).getBytes(StandardCharsets.UTF_8))
+            .withData(Objects.requireNonNull(JsonUtils.toJSONString(content)).getBytes(StandardCharsets.UTF_8))
```
Comment (mxsm) [inline eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java:196]: use Constants.DEFAULT_CHARSET replace StandardCharsets.UTF_8

File: eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java [hunk 2]
```diff
@@ -177,7 +178,7 @@ public static CloudEvent generateCloudEventV1Async() {
...
```
Comment (mxsm) [inline eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java:181]: use Constants.DEFAULT_CHARSET replace StandardCharsets.UTF_8

**Review by mxsm — COMMENTED** [review]
Hi @nikam14 you can create issue first and link this pr to issue

File: eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java (hunk 1 above)
Comment (nikam14) [reply eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java:196]: After replacing StandardCharsets.UTF_8 with Constants.DEFAULT_CHARSET Do I have make any other changes.

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**